c/zss.c

```diff
@@ -417,6 +418,38 @@ static JsonObject *readServerSettings(ShortLivedHeap *slh, const char *filename)
   return mvdSettingsJsonObject;
 }
 
+#define DEFAULT_CONFIG "                                               \
+                       {                                               \
+                         \"productDir\": \"../defaults\",              \
+                         \"siteDir\": \"../deploy/site\",              \ 
+                         \"instanceDir\": \"../deploy/instance\",      \
+                         \"groupsDir\": \"../deploy/instance/groups\", \
+                         \"usersDir\": \"../deploy/instance/users\",   \
+                         \"pluginsDir\": \"../defaults/plugins\",      \
+                         \"agent\": {},                                \
```

Comment:
this section is a little weird. if you're going to make defaults, the IP&port should be here too.

Reply:
Done

Reply:
Done